ci/sawtooth-build-debs

```diff
@@ -13,9 +13,17 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
-# USAGE:
-#   From sawtooth-core:
+# Description:
+#   Builds the environment needed to create the Sawtooth Lake python3 debs.
+#   Running the image will put the Sawtooth Lake debs in
+#   sawtooth-core/build/debs.
```

Comment:
Amend this to read "Running the image will put the Sawtooth Lake debs in sawtooth-core/build/debs on your local machine."